CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ a release.
 - Loggable: Remove unfixable deprecation when extending `LoggableListener`
 - Remove unfixable deprecations when extending repository classes
 - Fix error caused by the attempt of "doctrine/annotations" parsing a `@note` annotation
+- Fix bug collecting metadata for inherited mapped classes
```

Comment:
@magikid  This should go under `Unreleased` , currently it's under `[3.11.1] - 2023-02-20` which is wrong.

Reply:
Done, sorry, I was confused because there weren't any of the subheadings in the `Unreleased` section.